### PR TITLE
Update cookbook entries with best practices

### PR DESCRIPTION
**Pull request by JhonnyL**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets |  |

### CODE REVIEW

**Comment by javiereguiluz** [comment]
A big :+1: for this change! Thanks @JhonnyL for helping us getting rid of the old Acme bundle.

**Comment by xabbuh** [comment]
:+1:

**Comment by xabbuh** [comment]
@JhonnyL Did you make some changes? Looks like some rebase went wrong.

**Comment by JhonnyL** [comment]
@xabbuh Sorry, should be fixed now

**Comment by xabbuh** [comment]
Thank you @JhonnyL.

File: cookbook/form/data_transformers.rst [hunk 1]
```diff
@@ -161,17 +161,17 @@ to and from the issue number and the ``Issue`` object::
     namespace AppBundle\Form\DataTransformer;
 
     use AppBundle\Entity\Issue;
-    use Doctrine\Common\Persistence\EntityManager;
+    use Doctrine\Common\Persistence\ObjectManager;
```
Comment (WouterJ) [inline cookbook/form/data_transformers.rst]: the changes in this document seems not to be related to this PR (but to #5666 instead, which I just merged).

File: cookbook/form/direct_submit.rst [hunk 2]
```diff
@@ -28,7 +28,7 @@ submissions::
             return $this->redirect($this->generateUrl('task_success'));
         }
 
-        return $this->render('AcmeTaskBundle:Default:new.html.twig', array(
+        return $this->render('AppBundle:Default:new.html.twig', array(
```
Comment (WouterJ) [inline cookbook/form/direct_submit.rst]: this needs further fixing, as templates live in `app/Resources/views` with the best practices. This means you get `default/new.html.twig` (new standard is to use lowercase template paths).

This doesn't have to be fixed in this PR though, using AppBundle consistently is already a huge step forward.

**Pull request merged**